test/config/TestConfig.cpp

```diff
@@ -1026,6 +1099,86 @@ TEST_F(ConfigTestFixture, DeviceDefenderCli)
     ASSERT_EQ(6, config.deviceDefender.interval);
 }
 
+TEST_F(ConfigTestFixture, DeviceDefenderComparison)
```

Comment:
Seems like 3 of these configs are exactly the same? It isn't necessary to test like that. That would be like testing the code to see if its non-deterministic. Create a helper method that allows you to input some parameters to output a jsonString and test with different parameters and assert they are equal.